scalafix-cli/src/main/scala/scalafix/internal/v1/MainOps.scala

```diff
@@ -266,6 +266,8 @@ object MainOps {
         i += 1
       }
       val next = handleFile(args, file)
+      if (next != ExitStatus.Ok)
+        args.args.out.println(s"Unexpected error handling '$file', proceeding to the next.")
```

Comment:
Sure: it happens when applying the 'Collection213CrossCompat' rules to akka. Enabling scalafix [like this](https://github.com/akka/akka/compare/scalafix) and running `sbt 'akka-actor/scalafix Collection213CrossCompat'` yields https://pastebin.com/NJ4N9xs9 (sorry about all the escape codes that get misrendered there).

This left me in the dark as to which file failed, and whether the scalafix process was aborted or continued after that. Of course having better errors nearer to the root would also be helpful, but perhaps not always practical.

Reply:
Thanks for providing those details, very helpful.

I opened https://github.com/scalacenter/scalafix/pull/910 to improve handling of unexpected errors so that the message includes what file is relevant.